src/components/PostsList/index.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { loadPosts } from '../../store/posts/actions';
+import { selectPosts } from '../../store/posts/selectors';
+import './styles.scss';
+
+const PostsList = () => {
+	const dispatch = useDispatch();
+	const posts = useSelector(selectPosts);
+
+	useEffect(() => {
+		dispatch(loadPosts());
+	}, [dispatch]);
+
+	return (
+		<div className="wrapper">
+			{posts.map((post, index) => (
```

Comment:
лучше добавить проверку
```suggestion
			{posts.length > 0 && posts.map((post, index) => (
```